libraries/AP_Airspeed/AP_Airspeed.cpp

```diff
@@ -217,6 +221,15 @@ const AP_Param::GroupInfo AP_Airspeed::var_info[] = {
     // @Range: 0.0 10.0
     // @User: Advanced
     AP_GROUPINFO("_WIND_GATE", 26, AP_Airspeed, _wind_gate, 5.0f),
+    
+    // @Param: _MAX_SPD
+    // @DisplayName: Maximum offset cal speed error 
+    // @Description: The maximum speed change in airspeed reports that is allowed due to offset changes between calibraions before a warning is issued. This potential speed error is in percent of ASPD_FBW_MIN. 0 disables. Helps prevents calibrations without pitot being covered.
+    // @Description{Copter, Blimp, Rover, Sub}: This parameter and function is not used by this vehicle.
+    // @Range: 0.0 10.0
+    // @Units: %
+    // @User: Advanced
+    AP_GROUPINFO("_MAX_SPD", 27, AP_Airspeed, max_speed_pcnt, 0),    
```

Comment:
Can you use `AP_GROUPINFO_FRAME` and just specify only plane here? Then you only need a doc string for plane, and the other vehicles don't get any extra parameters.